README.md

```diff
@@ -181,6 +220,10 @@ create a .env file that includes the following:
 - PRISMA_MANAGEMENT_API_SECRET
 - PRISMA_ENDPOINT
 - PRISMA_SECRET
+- CODE_CLIMATE_API
+- CODE_CLIMATE_TOKEN
+- GIT_HUB_API
+- GIT_HUB_TOKEN
 
```

Comment:
Should `SENDGRID_API_KEY` be here also?

Reply:
It should, however we were updating the things External Integrations touched to prevent potential merge conflicts.  I've added all the variables in the following commit.

Reply:
Ah...makes sense! Thank you for that.